client/graph/schema.resolvers.go

```diff
@@ -6,17 +6,18 @@ package graph
 import (
 	"context"
 	"io"
-	"log"
 
 	"github.com/99designs/gqlgen/graphql"
 	"github.com/Shelex/grpc-go-demo/client/graph/factory"
 	"github.com/Shelex/grpc-go-demo/client/graph/generated"
 	"github.com/Shelex/grpc-go-demo/client/graph/model"
 	"github.com/Shelex/grpc-go-demo/proto"
+	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc/metadata"
 )
 
 func (r *mutationResolver) AddEmployee(ctx context.Context, employee model.EmployeeInput) (*model.Employee, error) {
+	log.WithField("employee", employee).Info("requesting AddEmployee")
```

Comment:
You might want to take a look at doing this centrally using grpc interceptors. An example is in the service-chassis :

https://github.com/90poe/service-chassis/blob/master/grpc/logginginterceptor.go#L29-L46